requirements.txt

```diff
@@ -1,5 +1,5 @@
 molo.surveys~=8.0.3
-molo.core==8.0.3
+molo.core~=8.0.4
```

Comment:
I'm not sure I agree with making this dependency variable.
My feeling is that with something as integral to the project as molo.core is we should probably be explicit about which version we're using.
That's a subjective opinion though so I won't hold the PR to it

Reply:
@KaitCrawford Jamie actually said that this will make the docker image build fail. So I removed it, your intuition was correct